how-to/how-to-use-kata-containers-with-acrn.md

```diff
@@ -25,8 +25,30 @@ Please refer to ACRN [documentation](https://projectacrn.github.io/latest/index.
 
 This document requires the presence of the ACRN hypervisor and Kata Containers on your system. Install using the instructions available through the following links:
 
-- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).
-- ACRN [software](https://projectacrn.github.io/latest/getting-started/apl-nuc.html#use-the-script-to-set-up-acrn-automatically) setup.
+- ACRN supported [Hardware](https://projectacrn.github.io/latest/hardware.html#supported-hardware).This guide was tested using KBL-NUC(NUC7i7DNH)
```

Comment:
- Missing space before period.
- Missing final period.
- What is "KBL-NUC"? tbh, I think this NUC information may be better as part of the commit rather than being part of the document.

Reply:
Removed tested hardware and will add it in the commit message.

>